whylog_vim/tests/tests_proxy/tests_parser_menu_operations.py

```diff
@@ -0,0 +1,30 @@
+import six
+from unittest import TestCase
+from . import create_mock_editor
+from whylog_vim.proxy import WhylogProxy
+from mock import patch
+
+
+class TeacherMenuTests(TestCase):
+    def setUp(self):
+        self.editor = create_mock_editor()
+        self.whylog_proxy = WhylogProxy(self.editor)
+        self.whylog_proxy.teach()
+        self.whylog_proxy.teach()
+
+    def tests_edit_content(self):
+        numb = next(six.iterkeys(self.whylog_proxy.teacher.output.buttons))
+        self.editor.get_line_number.return_value = numb
```

Comment:
Check if not `self.assertEqual(self.whylog_proxy.teacher.rule.parsers[0].line_content, 'some line')` otherwise we are not sure if it really changes.